### PR TITLE
fix: components/admin linting

### PR DESCRIPTION
**Pull request by nikrb**
fixed components/admin directory linting.
Two issues remain on AdminForm.js:
```
/home/nik/projects/chingu/voyage8/bears14/components/Admin/AdminForm.js
  427:13  error  A form label must be associated with a control                                      jsx-a11y/label-has-associated-control
  427:13  error  Form label must have ALL of the following types of associated control: nesting, id  jsx-a11y/label-has-for
```

### CODE REVIEW

File: components/Admin/AdminForm.js [hunk 1]
```diff
@@ -278,21 +282,38 @@ class AdminForm extends Component {
...
+      updating,
+      work
+    } = this.state;
 
     let workInfo = null;
 
-    if (this.state.work) {
-      const { name, collection, id } = this.state.work;
+    if (work) {
+      // TODO: check name, collection in state === work.name work.collection
```
Comment (nikrb) [inline components/Admin/AdminForm.js:307]: any help here? is it true to say `this.state.work.name === this.state.name && this.state.work.collection === this.state.collection` ?

Comment (g1st) [reply components/Admin/AdminForm.js:307]: No, not always. This ```this.state.work``` is being set just after item was updated or created.

Admin just logs on and goes to admin page to add new item:
 ```this.state.work = null```  and ```this.state.name = ''```.

Admin fills up form:
```this.state.work = null```  and ```this.state.name = 'Golden Ring'```.

Admin submits form:
```this.state.work = {id: 'abc123', name: 'Golden Ring', collection: 'flow'}``` and ```this.state.name = ''```.

After submitting we explicitly reset state (except ```this.state.work```) by calling ```resetForm()``` to clear form for another item to add. And update UI with just added item's name and collection. Probably need to change name of works property to something like ```updatedItem``` for clarity?

Comment (nikrb) [reply components/Admin/AdminForm.js:307]: thanx. I'll have another and keep them separate.

**Review by zurda — APPROVED** [review]

File: components/Admin/SignIn.js [hunk 2]
```diff
@@ -10,13 +10,12 @@ class Signin extends Component {
 
   handleSubmit = e => {
     e.preventDefault();
-    this.props.authenticate(
-      { email: this.state.email, password: this.state.password },
-      'signin'
-    );
+    const { email, password } = this.state;
+    authenticate({ email, password }, 'signin');
```
Comment (g1st) [inline components/Admin/SignIn.js:14]: Here we can't access imported action straight away. We dispatched it at the end of file in export function and here need to access it from props.

**Pull request merged**